src/js/panels.js

```diff
@@ -119,10 +119,176 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         that.locate("meter").css("height", percentage + "%");
     };
 
+    fluid.defaults("gpii.firstDiscovery.panel.keyboard", {
+        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        preferenceMap: {
+            "gpii.firstDiscovery.stickyKeys": {
+                "model.stickyKeys": "default"
+            }
+        },
+        selectors: {
+            input: ".gpiic-fd-keyboard-input",
+            instructions: ".gpiic-fd-keyboard-instructions",
+            assistance: ".gpiic-fd-keyboard-assistance"
+        },
+        selectorsToIgnore: ["assistance"],
+        events: {
+            onOfferAssistance: null,
+            onInitInput: null
+        },
+        model: {
+            // offerAssistance: boolean
+            // tryAccomodation: boolean
+            input: ""
+        },
+        components: {
+            assistance: {
+                type: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster",
+                createOnEvent: "onOfferAssistance",
+                container: "{that}.container",
+                options: {
+                    messageBase: "{keyboard}.options.messageBase",
+                    model: {
+                        tryAccomodation: "{keyboard}.model.tryAccomodation"
+                    },
+                    modelRelay: [{
+                        source: "stickyKeysEnabled",
+                        target: "{keyboard}.model.stickyKeys",
+                        forward: "liveOnly",
+                        singleTransform: {
+                            type: "fluid.transforms.identity"
+                        }
+                    }, {
+                        source: "stickyKeysEnabled",
+                        target: "{keyboardInput}.model.stickyKeysEnabled",
+                        backward: "liveOnly",
```

Comment:
It might be clearer to reverse the first of these relay rules. It seems that the model is that the initial value for this field is to be taken from the root component, and the other sources to be ignored. I think it would overall be clearer to use the full reference {gpii.firstDiscovery.panel.keyboard} rather than just simply {keyboard} which starts to run the risk of confusion (at least to human readers) with {keyboardInput}. The upcoming version of the framework will allow this reference to be written as {keyboard&panel} which might be better than either of these choices.